lib/data_pull.rb

```diff
@@ -198,6 +206,24 @@ def run(args:, config:)
         json: output,
       )
     end
+
+    private
+
+    def compute_requesting_issuers(uuids)
+      service_providers = ServiceProviderIdentity.where(uuid: uuids).pluck(:service_provider)
+      return nil if service_providers.empty?
+      service_provider = service_providers.tally.max_by { |_sp, count| count }[0]
```

Comment:
maybe use a destructure to name the array index clearer?
```suggestion
      service_provider, _count = service_providers.tally.max_by { |_sp, count| count }
```